### DISCUSSION TITLE
怎样关掉每行下面的横线

### DISCUSSION THREAD

**Discussion by Yan-jw20** [dail8859/NotepadNext#597] [General]
怎样关掉每行下面的横线，有点不习惯

**Comment by dail8859** [comment]
Please post a screenshot of the issue you are seeing.

**Reply by willmove** [reply]
<img width="1065" height="760" alt="image" src="https://github.com/user-attachments/assets/115f614b-8d0b-438e-9c6f-bc442d0bf695" />

**Reply by dail8859** [reply]
This is an issue that would need fixed by the Scintilla project. I asked them about this a while ago but nothing was implemented as far as I'm aware.
